scripts/getPackages.ts

```diff
@@ -1,8 +1,5 @@
 /* eslint-disable no-console */
```

Comment:
can we remove this script entirely. It seems redundent

Reply:
Are you sure? I will need to re-write / remove the entire **package-finder** job from workflows/test.yml ( as in [https://github.com/digicatapult/veritable-cloudagent/pull/20/files/df7b0948f4e245c0d6f991865a3a78f68a726bd3](https://github.com/digicatapult/veritable-cloudagent/pull/20/files/df7b0948f4e245c0d6f991865a3a78f68a726bd3#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R109-R111) )

Reply:
Leave it for now I'll do that when I integrate in. It doesn't break anything this is just a bit redundent